components/rmrk/Gallery/AvailableActions.vue

```diff
@@ -165,7 +161,10 @@ export default class AvailableActions extends mixins(
   }
 
   get showMeta() {
-    return ['SEND', 'LIST'].includes(this.selectedAction)
+    return !!(
```

Comment:
Not sure why you need `!!` as it just does Boolean conversion 


```js
x = true && [1].inlcudes(1)
// x will be always boolean
``

Reply:
1. `this.selectedAction` may be `null`,  so it can not be used in `Array.includes`
2. `showMeta() ` should return boolean, so if `this.selectedAction` was `null`, we need to convert it to `false`

Reply:
for the ts check